cli/command/formatter/stack.go

```diff
@@ -48,8 +51,9 @@ type stackContext struct {
 func newStackContext() *stackContext {
 	stackCtx := stackContext{}
 	stackCtx.header = map[string]string{
-		"Name":     nameHeader,
-		"Services": stackServicesHeader,
+		"Name":         nameHeader,
+		"Services":     stackServicesHeader,
+		"Orchestrator": stackOrchestrastorHeader,
```

Comment:
Can you:

- Add the new `Orchestrator` placeholder to the "formatting" section in the docs: https://github.com/docker/cli/blob/master/docs/reference/commandline/stack_ls.md#formatting
- Update the `Examples` section to show the new column?

Reply:
This should be fixed, can you PTAL? Thanks!